implementations/rust/ockam/ockam_command/src/message/send.rs

```diff
@@ -38,7 +38,7 @@ pub struct SendCommand {
     #[arg(long)]
     pub hex: bool,
 
-    /// Override Default Timeout
+    /// Override default timeout (in seconds)
```

Comment:
One finesse we could implement at some stage would be to have a value parser to `Duration` so that we can pass `10s`, `1m`, `500ms` on the command line

Reply:
That would be really nice! I'll create an issue for this.